aws/resource_aws_dynamodb_table_test.go

```diff
@@ -1186,6 +1247,24 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
 `, rName)
 }
 
+func testAccAWSDynamoDbBilling_PayPerRequest(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
+  name = "%s"
+  billing_mode = "PAY_PER_REQUEST"
+  hash_key = "TestTableHashKey"
+
+  attribute {
+    name = "TestTableHashKey"
+    type = "S"
+  }
+  point_in_time_recovery {
```

Comment:
Does `PAY_PER_REQUEST` require point in time recovery?

Reply:
@bflad `PAY_PER_REQUEST` does not require point in time recovery.
<img width="594" alt="dynamotablesettings" src="https://user-images.githubusercontent.com/10055646/49269380-87880c00-f419-11e8-8ec9-7d17fe505670.png">